config/networkpolicies/10-egress-policies.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-k8s-api-egress
```

Comment:
I believe, this policy should be a part of the bundle as this should be deployed by OLM not operator.